src/backend/gl/src/info.rs

```diff
@@ -171,6 +171,7 @@ pub struct PrivateCaps {
     pub buffer_storage_supported: bool,
     pub clear_buffer_supported: bool,
     pub frag_data_location_supported: bool,
+    pub is_embedded: bool,
```

Comment:
The first code link uses it correctly, since `Info` is not a capability struct.
The second one is indeed a bad example on our part, I'd prefer it to be fixed in the way I described here.